presto-tests/src/test/java/com/facebook/presto/tests/TestTpchQueriesSelectivityStats.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.tests.statistics.StatisticsAssertion;
+import com.facebook.presto.tpch.ColumnNaming;
+import com.facebook.presto.tpch.TpchConnectorFactory;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.SystemSessionProperties.USE_NEW_STATS_CALCULATOR;
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.defaultTolerance;
+import static com.facebook.presto.tests.statistics.Metrics.OUTPUT_ROW_COUNT;
+import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+
+/*
+The tests in this class have been written to ensure that the statistics code
+does not introduce regressions in selectivity estimates of table rows in TPCH
+queries in future. The enabled tests are the ones which pass in 0.179-t (i.e.
+the error in selectivity estimates for those queries is within the default
+tolerance range). The disabled tests fail in 0.179-t. They should be enabled
+later as the selectivity estimates improve over time.
+ */
+public class TestTpchQueriesSelectivityStats
```

Comment:
To me, it looks like the tests in `TestTpchLocalStats` are focused on testing stats for different functionalities such as union, aggregation etc. I don't know if they were written from the snippets of TPC-H queries. Also, I followed a different naming convention for the test names in this class, so that it will be easier for someone else to know the "coverage" of these tests (i.e. what all TPC-H queries' snippets are in here). So I thought it's best to have them in a separate class.